goentri.com.webflow.json

```diff
@@ -29,16 +29,23 @@
     {
       "type": "CNAME",
       "host": "www",
-      "pointsTo": "%cnamePointsTo%",
+      "pointsTo": "%proxySslUrl%",
       "groupId": "a1",
       "ttl": 3600
     },
     {
       "type": "TXT",
       "host": "@",
-      "data": "%txt%",
+      "data": "%proxySslUrl%",
```

Comment:
Sure about it with Url in TXT record?

Reply:
Hey @pawel-kow! yeah, that's what we need, thanks for double checking 🙏